data/data/powervs/variables-powervs.tf

```diff
@@ -40,6 +40,18 @@ variable "cloud_instance_id" {
     default     = ""
 }
 
+variable "cloud_plan" {
+    type        = string
+    description = "The name of the ibm cloud plan"
+    default     = ""
```

Comment:
Let's talk to the PowerVS team and find a suitable default plan we use for regular PowerVS service instance creation.

Reply:
This should be solved.